lib/src/core/themes/custom_themes/text_field_theme.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_clean_architecture/src/core/styles/app_colors.dart';
+
+class TTextFormFieldTheme {
+  TTextFormFieldTheme._(); // to avoid creating instances
+
+  /* Light Theme */
+  static InputDecorationTheme lightInputDecorationTheme = InputDecorationTheme(
```

Comment:
same here create base style for InputDecorationTheme and use it for both light and dark